lib/pg.ddl.js

```diff
@@ -486,21 +484,42 @@ const generateTables = (categories, types) => {
     .join('\n');
 };
 
-const createHistorySchema = schema => {
+const createHistorySchema = (schema, domains, categories) => {
   const history = Object.assign({}, schema);
-  history._Creation = { domain: 'DateTime', required: true, index: true };
-  history._Effective = { domain: 'DateTime', required: true, index: true };
-  history._Cancel = { domain: 'DateTime', index: true };
-  history._HistoryStatus = { domain: 'HistoryStatus', required: true };
+  history._Creation = {
+    domain: 'DateTime',
+    required: true,
+    index: true,
+    definition: domains.get('DateTime'),
```

Comment:
I think you can save the value of `domains.get('DateTime')` into the variable to avoid calling `get()` multiple times here.